### PR TITLE
Publish 95 new BOD System

### PR DESCRIPTION
**Pull request by kevin-10**
- Added Publish 95 new BOD system: 
https://uo.com/wiki/ultima-online-wiki/gameplay/crafting/bulk-orders/

- System includes some long overdue overhaul of craftables that can be crafted with resources, and made exceptional, see DefCooking and DefTinkering.

- New system, by default, is true if Core.TOL.  If set to false, it will use the old BOD system.

- All items should have the proper functionality, as all content has been tested and verified on an EA account as well as uo.com.

- Updated Body Table with new mobiles
- Fixed indentation in Regions.xml, not that it was broken.

### CODE REVIEW

**Comment by kevin-10** [comment]
This is a pretty big PR.  I tested it pretty vigorously for accuracy to EA.

File: Scripts/Gumps/BaseRewardGump.cs [hunk 1]
```diff
@@ -93,7 +102,7 @@ public void DisplayRewardPage()
                 CollectionItem item = Collection[Index];
 				
 				int height = Math.Max(item.Height, 20);
-
+                Console.WriteLine("POints2: {0}", Points);
```
Comment (kevin-10) [inline Scripts/Gumps/BaseRewardGump.cs]: Haha, yea, I saw that earlier.

**Comment by Tasanar** [comment]
I played around with it for a while. Looks good and no issues reported so far.

**Comment by kevin-10** [comment]
Added a crash fix which shouldnt happen very often.  Also added an exploit fix that is a pretty bad one if anyone has figured it out.

**Comment by ppardalj** [comment]
@kevin-10 is the exploit related somehow with the branch content? If not, it would be a good idea to push it separately in another branch so it's not blocked by the review of the new bod system and gets into master asap.

**Comment by kevin-10** [comment]
No, it's from an existing system.  I'm not sure how to do the branch thing, I'll have to figure that out next time.

**Comment by Tasanar** [comment]
Is it the reforge and imbue scripts we should update then? From this pull request? Would want to do this asap as well.

**Comment by kevin-10** [comment]
Yes.  I was PM'd by an awesome member on the forums about an expoloit.  Once I verified it, I pushed it to this commit.  Since this was the first I've heard of it, there doesn't appear to be anyone abusing this that we know of.  *crosses fingers*

Tasanar, have you been actively running the new bod system on your server?

**Comment by Tasanar** [comment]
No sir I was just testing it myself on my test shard. I am going to push it today though to also fix the pointed out exploit. I have messed with it for hours though and everything seems to work well.

After it is posted to the main repo could you please explain what players were able to do? That way I can do some searching to make sure it was not exploited on my shard through [xmlfind.

**Comment by kevin-10** [comment]
I'll PM you.

**Comment by ppardalj** [comment]
@kevin-10 if you need to create a separate pull request for concrete changes, it's very easy, just use the following commands:
```
$ git checkout master
$ git checkout -b your-new-branch-name
# develop your fixes
$ git commit -m "Fixed blah"
$ git push origin your-new-branch-name
# Create a PR for the new branch
```

But please stop push fixes to a feature branch, it makes baby Jesus cry :(

**Comment by kevin-10** [comment]
Yea I know!  Can that be done from the GUI?  I have never used it with commands.

**Comment by ppardalj** [comment]
@kevin-10 it depends on the UI you're using, but most of them should support it, they're pretty basic operations. Please check the documentation of the tool you're using. If you tell us which one you're using, maybe we can try to help.

**Comment by kevin-10** [comment]
I think its tortoise git.

**Comment by ppardalj** [comment]
@kevin-10  take a look at this https://tortoisegit.org/docs/tortoisegit/tgit-dug-branchtag.html

**Comment by kevin-10** [comment]
closed due to technical difficulties.

**Pull request closed without merging**